sagemaker-python-sdk/tensorflow_serving_using_elastic_inference_with_your_own_model/tensorflow_neo_compiled_model_elastic_inference.ipynb

```diff
@@ -0,0 +1,227 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Amazon Elastic Inference with Neo-compiled TensorFlow model on SageMaker\n",
+    "\n",
+    "This notebook demonstrates how to compile a pre-trained TensorFlow model using Amazon SageMaker Neo, and deploy this compiled model on SageMaker with Elastic Inference.\n",
```

Comment:
Possible re-wording (feel free to modify):
"This notebook demonstrates how to compile a pre-trained TensorFlow model using Amazon SageMaker Neo and how to deploy this model to a SageMaker Endpoint with Elastic Inference"